tests/testcontainers/es.py

```diff
@@ -209,3 +209,9 @@ def index(self, **kwargs: Any) -> dict[str, Any]:
             self._index_indices.add(kwargs["index"])
```

Comment:
This file also has `_DEFAULT_VERSION = "7.17.20"` 
in [here](https://github.com/elastic/elastic-serverless-forwarder/blob/b4e1c8fb3335f6a10173e77f15b06b1e3bd8234d/tests/testcontainers/es.py#L34)

Can we update this as well to 8.14?

Reply:
I'd rather not include upgrading the testing infrastructure as part of this PR since the logic here is not strictly dependant on ES version. ESF is still using [elasticsearch client 7.17 ](https://github.com/elastic/elastic-serverless-forwarder/blob/7025566921f61aeccf4e34c67fa07a17b2fe93b7/requirements.txt#L4) so it matches the lowest supported version as is.